.github/CODEOWNERS

```diff
@@ -1,1427 +1,1852 @@
 # Instructions for CODEOWNERS file format and automatic build failure notifications:
+
```

Comment:
There's a lot of blanks that seem unintentional.   Can we revert those, please?

Reply:
Hmm. Not sure how they got there. Maybe VS Code?

Reply:
Fixed. And it was VS code.